packages/http/src/router/index.ts

```diff
@@ -2,10 +2,74 @@ import { types } from '@stoplight/prism-core';
 import { IHttpOperation } from '@stoplight/types';
 
 import * as t from '../types';
+import { matchPath } from './matchPath';
+import { IMatch, MatchType } from './types';
+import { matchBaseUrl } from './matchBaseUrl';
 
 export const router: types.IRouter<IHttpOperation, t.IHttpRequest, t.IHttpConfig> = {
   route: async ({ resources, input, config }) => {
-    // given input and config, find and return the matching resource
-    throw new Error('Method not implemented.');
+    return Promise.resolve(route(resources, input));
   },
 };
+
+function route(resources: IHttpOperation[], request: t.IHttpRequest) {
```

Comment:
Is there a need to keep it as a separate func?
Could be moved to public route() method.